src/test/java/ch/uzh/ifi/hase/soprafs22/service/UserServiceTest.java

```diff
@@ -68,7 +135,20 @@ void createUser_duplicateInputs_throwsException() {
 
     // then -> attempt to create second user with same user -> check that an error
     // is thrown
-    assertThrows(ResponseStatusException.class, () -> userService.createUser(testUser));
+    ResponseStatusException e = assertThrows(ResponseStatusException.class, () -> userService.createUser(testUser));
+    assertEquals(HttpStatus.CONFLICT, e.getStatus());
+  }
+
+  @Test
+  void logOutUser_success(){
+      long id = testUser.getId();
+        //when -> looking for testuser by id then return testUser
+      Mockito.when(userRepository.findById(id)).thenReturn(testUser);
```

Comment:
Why don't you use Mockito.any here?

Reply:
In this particular case it seems that nothing else works.. It is rather weird..